andela/ah-orcas
# #161254664 Rate an article

#### What does this PR do?
```registered user can rate an article ```
#### Description of Task to be completed?
```Our community as authors is built on trust. Rating each other's articles helps to gain and build trust. Leaving a rating is your chance to give other authors a general idea of what we feel about their content. ```
#### How should this be manually tested?
- Clone this repo.
``` 
$ git clone https://github.com/andela/ah-orcas.git
```
- Navigate to this branch.
``` 
$ git clone git checkout  ft-reset-password-161254660
```
- create a python virtual environment in your local machine and start it.
```code
$ virtualenv env
$ source env/bin/activate
```
- Create a .env file and run it. Check sample.env file to configure yours.
```
$ source .env
```
- Run the app
```
$ python manage.py runserver
```
- Test the endpoint.
```
api/users/article/<str:slug>/
```

#### Any background context you want to provide?
```I created a dummy article model for testing ```
#### What are the relevant pivotal tracker stories?
 [#161254664](https://www.pivotaltracker.com/story/show/161254664)
